### ISSUE TITLE
Change over Time and Difference columns are blank in Census (Decennial) tab

### ISSUE THREAD

**Issue by pichot** [NYCPlanning/labs-factfinder#736]
![NYC_Population_FactFinder](https://user-images.githubusercontent.com/848177/68708063-f09c5d00-0560-11ea-9968-64b6171f4e98.jpg)

**Comment by godfreyyeung** [comment]
@allthesignals Do you think there's a simple fix for this?

**Comment by allthesignals** [comment]
Definitely an API issue, not seeing the change values come through.

**Comment by allthesignals** [comment]
This is probably not a simple fix but happy to look at it.

**Comment by allthesignals** [comment]
Our code was skipping change/difference calculations when reliability/significance statistics were missing or null in the row. However, change/sum calculations may still be performed on decennial. Our code wasn't taking this into account.

**Comment by allthesignals** [comment]
https://github.com/NYCPlanning/labs-factfinder-api/pull/95 is merged, but it only gets is part of the way there.

@EricaMaurer more changes coming down the pike, I want to make sure that I'm not mistaken: decennial profiles definitely get percentage change, percentage point change calculations, right?

**Comment by allthesignals** [comment]
Deployed changes on staging that calculate difference/change, but not their respective percents: https://factfinder-staging.planninglabs.nyc/profile/304/census

**Comment by allthesignals** [comment]
There are 4 functions that are relevant here, and have some pretty similar behavior:
calculateChangePercents
calculateChangePercentagePoints
calculateDifferences
calculateDifferencePercents

Both change/difference utility files follow a similar structure. Not sure what the better "abstraction" would be yet, but there is a lot of confusing logical "forking" going on in the code...

**Comment by EricaMaurer** [comment]
yup-- 
2010 census: number change and percentage point change
change over time (2000 to 2010): number change, percent change, and percentage point change

**Comment by allthesignals** [comment]
@EricaMaurer I've deployed changes that include handling for decennial change over time, etc.

What was happening before was that the code wasn't even calculating those things before it wasn't seeing any reliability/significance data. I made changes that allowed decennial to calculate.

I'm seeing the data come through for change and difference on staging. Can I get the go-ahead to close this issue? We can open up any downstream problems in a separate issue.
